data/refresh/rebuild_schedule_e.sql

```diff
@@ -1,31 +1,437 @@
 -- Create Schedule E table
 drop table if exists ofec_sched_e_tmp;
-create table ofec_sched_e_tmp as
-select
-    *,
-    cast(null as timestamp) as timestamp,
-    image_pdf_url(image_num) as pdf_url,
-    coalesce(rpt_tp, '') in ('24', '48') as is_notice,
-    to_tsvector(pye_nm) as payee_name_text,
-    now() as pg_date
-from fec_fitem_sched_e_vw;
+create table ofec_sched_e_tmp 
+(
+  cmte_id varchar(9),
+  cmte_nm varchar(200),
+  pye_nm varchar(200),
+  payee_l_nm varchar(30),
+  payee_f_nm varchar(20),
+  payee_m_nm varchar(20),
+  payee_prefix varchar(10),
+  payee_suffix varchar(10),
+  pye_st1 varchar(34),
+  pye_st2 varchar(34),
+  pye_city varchar(30),
+  pye_st varchar(2),
+  pye_zip varchar(9),
+  entity_tp varchar(3),
+  entity_tp_desc varchar(50),
+  exp_desc varchar(100),
+  catg_cd varchar(3),
+  catg_cd_desc varchar(40),
+  s_o_cand_id varchar(9),
+  s_o_cand_nm varchar(90),
+  s_o_cand_nm_first varchar(38),
+  s_o_cand_nm_last varchar(38),
+  s_o_cand_m_nm varchar(20),
+  s_o_cand_prefix varchar(10),
+  s_o_cand_suffix varchar(10),
+  s_o_cand_office varchar(1),
+  s_o_cand_office_desc varchar(20),
+  s_o_cand_office_st varchar(2),
+  s_o_cand_office_st_desc varchar(20),
+  s_o_cand_office_district varchar(2),
+  s_o_ind varchar(3),
+  s_o_ind_desc varchar(50),
+  election_tp varchar(5),
+  fec_election_tp_desc varchar(20),
+  cal_ytd_ofc_sought numeric(14,2),
+  dissem_dt timestamp without time zone,
+  exp_amt numeric(14,2),
+  exp_dt timestamp without time zone,
+  exp_tp varchar(3),
+  exp_tp_desc varchar(90),
+  memo_cd varchar(1),
+  memo_cd_desc varchar(50),
+  memo_text varchar(100),
+  conduit_cmte_id varchar(9),
+  conduit_cmte_nm varchar(200),
+  conduit_cmte_st1 varchar(34),
+  conduit_cmte_st2 varchar(34),
+  conduit_cmte_city varchar(30),
+  conduit_cmte_st varchar(2),
+  conduit_cmte_zip varchar(9),
+  indt_sign_nm varchar(90),
+  indt_sign_dt timestamp without time zone,
+  notary_sign_nm varchar(90),
+  notary_sign_dt timestamp without time zone,
+  notary_commission_exprtn_dt timestamp without time zone,
+  filer_l_nm varchar(30),
+  filer_f_nm varchar(20),
+  filer_m_nm varchar(20),
+  filer_prefix varchar(10),
+  filer_suffix varchar(10),
+  action_cd varchar(1),
+  action_cd_desc varchar(15),
+  tran_id varchar,
+  back_ref_tran_id varchar,
+  back_ref_sched_nm varchar,
+  schedule_type varchar(8),
+  schedule_type_desc varchar(90),
+  line_num varchar(12),
+  image_num varchar(18),
+  file_num numeric(7,0),
+  link_id numeric(19,0),
+  orig_sub_id numeric(19,0),
+  sub_id numeric(19,0) NOT NULL,
+  filing_form varchar(8),
+  rpt_tp varchar(3),
+  rpt_yr numeric(4,0),
+  election_cycle numeric,
+  "timestamp" timestamp without time zone,
```

Comment:
Can we get rid of this column? We only seem to be storing `NULL` in it.

Reply:
The timestamp column was added by 18F, I assume probably was planned to be used by the queue process.  My understanding is that now sched_e is not using the queue process so it is always null.  Is this correct?  If that is the case, I will drop this column.  Please let me know.

I had also checked the itemized.py in models, it seems scheduleE model does not refer to this column.

Reply:
Unfortunately it is also a poor name for a column, making it difficult to search in code.

Reply:
I agree.  Also usually I will try not to name a column with a reserved word...

Alternately, I can skip this column in the insert statement for now.  At least it will save us from doing the cast(null as timestamp) conversion in vain.  Thoughts?